data/gui/themes/default/window/help_browser.cfg

```diff
@@ -247,10 +268,9 @@
 															vertical_grow = true
 
 															[rich_label]
-																width = 800
+																width = 700
```

Comment:
What causes the shrink?

Reply:
A workaround for the text squishing issue mentioned in #9417. Apparent that only happens when the size allocated by the container is less than rich_label's `width` key and most likely in pages with tables. This is only temporary, until I find a better fix.